gio/src/subclass/application.rs

```diff
@@ -455,6 +451,10 @@ mod tests {
         @implements crate::Application;
     }
 
+    // NOTE: on windows arge and argv are ignored, even if we pass the arguments explicitly.
+    //
+    // Source: https://gitlab.gnome.org/GNOME/glib/-/blob/e64a93269d09302d7a4facbc164b7fe9c2ad0836/gio/gapplication.c#L2513-2515
+    #[cfg(not(target_os = "windows"))]
```

Comment:
Can you move that to only the failing comparison? That line where the args are checked?

It would seem useful to have the whole other parts of the `GApplication` subclassing run on Windows too

Reply:
Done